client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -499,6 +499,7 @@ export const ApplicationsTable: React.FC = () => {
           { value: "yellow", label: "Medium" },
           { value: "red", label: "High" },
           { value: "unknown", label: "Unknown" },
+          { value: "unassessed", label: "Unassessed" },
         ],
         getItemValue: (item) => item.risk ?? "",
```

Comment:
it seems the problem is caused by unknown value send from the server - we could add some extra protection here to prevent this happening in the future

Reply:
It is also confusing that a risk of "Unknown" is an actual valid thing to pick

And "Unassessed" just means the assessments have not yet been completed.

Both are actual valid distinct values.

Reply:
I added normalize code to the current version to make sure we only filter match on valid and known Risk type values.